tests/template/test_spreadsheet_builder.py

```diff
@@ -77,17 +76,25 @@ def test_user_friendly(self):
             print("expected: " + user_friendly_dict[key])
             self.assertEqual(user_friendly_dict[key], uf)
 
+
     # TODO improve this test, at the moment just tests that the spreadsheet can be built, nothing about the
     #  contents.
     def test_vanilla_spreadsheet(self):
-        file = "uf_test.xlsx"
-        template = SchemaTemplate()
+        file = "vanilla_test.xlsx"
+        template = SchemaTemplate(
+            metadata_schema_urls=["https://schema.humancellatlas.org/type/biomaterial/15.5.0/donor_organism",
+                                 "https://schema.humancellatlas.org/type/biomaterial/10.4.0/specimen_from_organism",
+                                 "https://schema.humancellatlas.org/type/biomaterial/13.3.0/cell_suspension",
+                                 "https://schema.humancellatlas.org/type/protocol/sequencing/6.2.0/library_preparation_protocol",
+                                 "https://schema.humancellatlas.org/type/file/9.2.0/sequence_file"]
+        )
```

Comment:
Are the changes here just for testing? yeah i see a TODO comment about improving the tests so if that's out of scope for this PR, this is fine.